src/storage/src/hummock/utils.rs

```diff
@@ -38,7 +38,7 @@ use crate::store::{ReadOptions, StateStoreRead};
 pub fn range_overlap<R, B>(
     search_key_range: &R,
     inclusive_start_key: &B,
-    inclusive_end_key: &B,
+    end_key: &Bound<&B>,
```

Comment:
`Bound<&B>` is `Copy`. We don't need to pass a reference to it.

Reply:
> `Bound<&B>` is `Copy`. We don't need to pass a reference to it.

fixed